### PR TITLE
[communication] Live Testing - Introduce CloudConfig into test.yml 

### PR DESCRIPTION
**Pull request by beltr0n**
This modifies how we interact with the build pipeline to populate env vars

### CODE REVIEW

File: sdk/communication/tests.yml [hunk 1]
```diff
@@ -23,11 +23,10 @@ stages:
         ServiceDirectory: communication
         DeployArmTemplate: true
         TestSamples: true
-        EnvVars:
-          AZURE_TEST_RUN_LIVE: 'true'
```
Comment (minnieliu) [inline sdk/communication/tests.yml:27]: Can you get rid of the `AZURE_TEST_RUN_LIVE` as well?

Comment (beltr0n) [reply sdk/communication/tests.yml:27]: I'll add that to the python config since this is python only :)

Comment (beltr0n) [reply sdk/communication/tests.yml:27]: also need to add AZURE_COMMUNICATION_SERVICE_CONNECTION_STRING

**Review by lsundaralingam — APPROVED** [review]
LGTM!

File: sdk/communication/test-resources.json [hunk 2]
```diff
@@ -13,11 +13,37 @@
             "defaultValue": "communication",
             "type": "string"
         },
+        "communicationServicesEndpointSuffix": {
+            "defaultValue": ".communication.azure.com",
+            "type": "string"
+        },
+        "subscriptionId": {
```
Comment (sacheu) [inline sdk/communication/test-resources.json:20]: I think subscriptoin id is not needed. I have a PR which fixes this in .net.

File: sdk/communication/test-resources.json [hunk 3]
```diff
@@ -46,9 +72,29 @@
         }
     ],
     "outputs": {
+        "AZURE_SUBSCRIPTION_ID": {
```
Comment (sacheu) [inline sdk/communication/test-resources.json:75]: Do we need this? I think .net work without it.

File: sdk/communication/test-resources.json [hunk 4]
```diff
@@ -46,9 +72,29 @@
...
+        "AZURE_CLIENT_SECRET": {
+            "type": "string",
+            "value": "[parameters('testApplicationSecret')]"
+        },
         "COMMUNICATION_CONNECTION_STRING": {
             "type": "string",
             "value": "[listKeys(resourceId('Microsoft.Communication/CommunicationServices',variables('uniqueSubDomainName')), '2020-08-20-preview').primaryConnectionString]"
-        }
+        },
+        "COMMUNICATION_SERVICE_ENDPOINT": {
```
Comment (sacheu) [inline sdk/communication/test-resources.json:95]: Do we need this? I think .net work without it.

File: sdk/communication/test-resources.json [hunk 5]
```diff
@@ -13,11 +13,37 @@
             "defaultValue": "communication",
             "type": "string"
         },
+        "communicationServicesEndpointSuffix": {
```
Comment (sacheu) [inline sdk/communication/test-resources.json:16]: Do we need this? I think .net work without it.

File: sdk/communication/tests.yml (hunk 1 above)
Comment (sacheu) [inline sdk/communication/tests.yml:27]: I think we need this, right?

**Review by sacheu — CHANGES_REQUESTED** [review]

File: sdk/communication/test-resources.json (hunk 3 above)
Comment (beltr0n) [reply sdk/communication/test-resources.json:75]: Well, Josh removed it last time from JS and it started failing. Let's sync up on your .net fix

File: sdk/communication/tests.yml (hunk 1 above)
Comment (beltr0n) [reply sdk/communication/tests.yml:27]: This has been put into the python subconfig

**Review by sacheu — APPROVED** [review]
Looks good. Thanks Bert

**Pull request merged**